src/keystone/real_test.mak

```diff
@@ -0,0 +1,141 @@
+# #############################################################################
+# File: src/keystone/shim_test.mak
```

Comment:
Same question about this make file: Are we ready, is it something that we an include into CI build.yml, so this make step gets executed every time?

If it's premature to test this build instruction as part of CI, that's fine. Just asking to evaluate if this can be included for CI-coverage.